### PR TITLE
Add a requirements section to the installation documentation.

### PR DESCRIPTION
**Pull request by tomzx**
- Specify that melody depends on composer

### CODE REVIEW

File: doc/install.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+------------
+
+Melody depends on [composer](https://getcomposer.org/) to install packages. To do so, it'll need to find an available instance of composer on your machine. It will search for `composer` and `composer.phar` in the current directory as well as in your path.
+
 Locally
 -------
 
-Download the [melody.phar](http://get.sensiolabs.org/melody.phar) file and store
-it somewhere on your computer.
+Download the [melody.phar](http://get.sensiolabs.org/melody.phar) file and store it somewhere on your computer.
```
Comment (lyrixx) [inline doc/install.md]: I always wrap text at 80 columns on purpose. Could you revert this change and also wrap the new chapter you added ? Thanks

**Comment by tomzx** [comment]
Updated per your request.

**Comment by lyrixx** [comment]
Thanks @tomzx.
